src/test/javascript/stringExtensionSpec.js

```diff
@@ -49,8 +49,8 @@ define(['bytepushers'], function (BytePushers) {
                 expect(result).toBe(false);
             });
 
-            it('does this string include play', function () {
-                var result = testStr.includes("play");
+            it('does this string include counter', function () {
+                var result = testStr.includes("counter");
```

Comment:
The referenced string could be rewritten to make more understandable or add a comment on the unit test.